min_oe_openMM.py

```diff
@@ -375,7 +361,7 @@ def load_and_minimize(infiles, log, output_dir,
                 os.makedirs(os.path.join(output_dir, 'GAFF'))
 
             fulln = os.path.join(output_dir+'/GAFF', fname)
-            log.write('Starting on GAFF optimization for %s\n'\
+            print('Starting on GAFF optimization for %s\n'\
```

Comment:
Did you mean to change this back to print where the other statements are written to log?